src/utils/validations/AdrValidation.ts

```diff
@@ -14,33 +14,33 @@ const tc3Types: string[] = [
 
 export const adrValidation = Joi.object().keys({
   vehicleDetails: Joi.object().keys({
-    type: Joi.string().required(),
-    approvalDate: Joi.date().format("YYYY-MM-DD").raw().required()
+    type: Joi.string().required().allow(null, ''),
+    approvalDate: Joi.date().format("YYYY-MM-DD").raw().required().allow(null, '')
   }).required(),
-  permittedDangerousGoods: Joi.array().items(Joi.string()).min(1).required(),
-  compatibilityGroupJ: Joi.boolean().optional().allow(null),
-  additionalExaminerNotes: Joi.string().max(1024).optional().allow(null),
+  permittedDangerousGoods: Joi.array().items(Joi.string()).min(1).required().allow(null, ''),
+  compatibilityGroupJ: Joi.boolean().optional().allow(null).allow(null, ''),
+  additionalExaminerNotes: Joi.string().max(1024).optional().allow(null).allow(null, ''),
   applicantDetails: Joi.object().keys({
-    name: Joi.string().max(150).required(),
-    street: Joi.string().max(150).required(),
-    town: Joi.string().max(100).required(),
-    city: Joi.string().max(100).required(),
-    postcode: Joi.string().max(25).required()
+    name: Joi.string().max(150).required().allow(null, ''),
+    street: Joi.string().max(150).required().allow(null, ''),
+    town: Joi.string().max(100).required().allow(null, ''),
+    city: Joi.string().max(100).required().allow(null, ''),
+    postcode: Joi.string().max(25).required().allow(null, '')
   }).required(),
   memosApply: Joi.array().items(Joi.string().allow(null)).optional().allow(null),
   documents: Joi.array().items(Joi.string().allow(null)).optional().allow(null),
   listStatementApplicable: Joi.boolean().optional().allow(null),
```

Comment:
```suggestion
  memosApply: Joi.array().items(Joi.string().allow(null, "")).optional().allow(null),
  documents: Joi.array().items(Joi.string().allow(null, "")).optional().allow(null),
  listStatementApplicable: Joi.boolean().optional().allow(null),
```
don't know how much it matters